core/arch/arm/plat-imx/registers/imx8m.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2019 NXP
+ *
```

Comment:
minor: empty line

Reply:
Addressed in 03c1e4790198c7f0f064b28447ef8ce0e637376e